src/index.ts

```diff
@@ -46,217 +47,222 @@ app.use((req, res, next) => {
 const provider = getProvider();
 const { ovalConfigs } = env;
 
-// Start restful API server to listen for root inbound post requests.
-app.post("/", async (req, res, next) => {
-  try {
-    const { url, method, body } = req;
-    Logger.debug(req.transactionId, `Received: ${method} ${url}`, { body });
+(async () => {
+  const keyManager = PrivateKeyManager.getInstance();
+  await keyManager.initialize(ovalConfigs);
```

Comment:
Most of the changes in this file are due to indentation after adding this at the beginning

Reply:
is the whole async wrapper required just for this line?  seems like this could be avoided. it does not need to be initialized exactly before the routes are set up, ideally yes, but the routes could also throw an error until this is fully initialized, removing the async wrapper

Reply:
That's great feedback. Agree, the async wrapper was overkill and what you suggest instead makes sense. It shouldn't take more than a minute to fetch all the keys. Fixed